stubs/invoke/invoke/terminals.pyi

```diff
@@ -0,0 +1,11 @@
+from io import TextIOWrapper
+from typing import Iterator
+
+WINDOWS: bool
+
+def pty_size() -> tuple[int, int]: ...
+def stdin_is_foregrounded_tty(stream: TextIOWrapper) -> bool: ...
```

Comment:
Do we really need to use the concrete TextIOWrapper type here? Generally we try to use narrow protocols for IO.

Reply:
I tried to use IO, but somehow it didn't work. Can you please support?

Reply:
This function only tries to access `fileno()`, but works even if `fileno()` is not supported. I suggest to use `Any` here:

```suggestion
def stdin_is_foregrounded_tty(stream: Any) -> bool: ...
```